src/TableFunctions/ITableFunctionFileLike.cpp

```diff
@@ -38,16 +49,16 @@ void ITableFunctionFileLike::parseArguments(const ASTPtr & ast_function, Context
     if (args.empty())
         throw Exception("Table function '" + getName() + "' requires at least 1 argument", ErrorCodes::NUMBER_OF_ARGUMENTS_DOESNT_MATCH);
 
-    for (auto & arg : args)
-        arg = evaluateConstantExpressionOrIdentifierAsLiteral(arg, context);
+    parseFirstArguments(args[0], context);
 
-    filename = args[0]->as<ASTLiteral &>().value.safeGet<String>();
+    for (size_t i = 1; i < args.size(); ++i)
+        args[i] = evaluateConstantExpressionOrIdentifierAsLiteral(args[i], context);
```

Comment:
Why don't we evaluate identifiers as literals for file table function with file descriptors? It's supposed that such queries should work:
`clickhouse local -q "select * from file(data.jsonl)"`
But now it throws an exception `Unknown identifier 'data.jsonl' in first arguments.`